scss/components/_prism.scss

```diff
@@ -77,3 +77,8 @@ code.language-markdown {
   padding-left: 18px;
   color: #fff;
 }
+
+.code-toolbar pre[class*="language-sh"] {
+  z-index: 1000000;
```

Comment:
`z-index: 10` has the same outcome. Also, a code comment as to why this is here would be really helpful.